### PR TITLE
CTCP-2151: AddShippingMark yes no page

### PR DESCRIPTION
**Pull request by kevincoatsworth**

### CODE REVIEW

File: app/utils/cyaHelpers/SummaryListRowHelper.scala [hunk 1]
```diff
@@ -55,6 +55,8 @@ private[utils] class SummaryListRowHelper(implicit messages: Messages) {
 
   protected def formatAsCountry(country: Country): Content = country.description.toText
 
+  protected def formatAsPackage(packageType: PackageType): Content = packageType.toString.toText
```
Comment (jackheslop96) [inline app/utils/cyaHelpers/SummaryListRowHelper.scala:58]: Does this not just do the same thing as `formatAsText`?

Comment (kevincoatsworth) [reply app/utils/cyaHelpers/SummaryListRowHelper.scala:58]: Yea they passed, I'll switch and push up

File: app/utils/cyaHelpers/SummaryListRowHelper.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 package utils.cyaHelpers
 
-import models.reference.Country
+import models.reference.{Country, PackageType}
```
Comment (jackheslop96) [inline app/utils/cyaHelpers/SummaryListRowHelper.scala:19]: Unused import?

Comment (kevincoatsworth) [reply app/utils/cyaHelpers/SummaryListRowHelper.scala:19]: Yea didn't spot that when I removed the formatAsPackage function!

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**